fiatconnect-api.md

```diff
@@ -2,245 +2,255 @@
 
 # 0. Table of Contents
 
+- [FiatConnect CICO Provider API Specification "RFC"](#fiatconnect-cico-provider-api-specification-rfc)
```

Comment:
Generated this using a vscode md plugin which added this line too. Can remove if it is confusing.